packages/connect-react/src/hooks/form-context.tsx

```diff
@@ -395,6 +436,98 @@ export const FormContextProvider = <T extends ConfigurableProps>({
     checkPropsNeedConfiguring()
   };
 
+  const handleSdkErrors = (o: unknown[] | unknown | undefined) => {
+    let newErrors = [
+      ...sdkErrors,
+    ]
+    if (!o) return
+    const handleObservationErrors = (observation: never) => {
+      const name: string = observation.err?.name
+      const message: string = observation.err?.message
+      if (name && message) return {
+        name,
+        message,
+      } as Record<string, string>
+    }
+
+    if (Array.isArray(o) && o.length > 0) {
+      for (let i = 0; i < o.length; i++) {
+        const item = o[i]
+        if (typeof item === "string") {
+          try {
+            const json = JSON.parse(item)
+            const name = json.name
+            const message = json.message
+            if (name && message) {
+              newErrors.push({
+                name,
+                message,
+              } as Record<string, string>)
```

Comment:
Can we define some types for these? `Record<string, string>` is extremely generic. We have a package called `@pipedream/types` (within the `types/` directory in this repo) that seems like a good place to define it, but we can start defining it internally in this package instead.